utilities/common.py

```diff
@@ -160,6 +167,9 @@ def common_parser_combine():
     parser.add_argument("--skipSignalSystOnFakes" , action="store_true", help="Do not propagate signal uncertainties on fakes, mainly for checks.")
     parser.add_argument("--noQCDscaleFakes", action="store_true",   help="Do not apply QCd scale uncertainties on fakes, mainly for debugging")
     parser.add_argument("--doStatOnly", action="store_true", default=False, help="Set up fit to get stat-only uncertainty (currently combinetf with -S 0 doesn't work)")
+    parser.add_argument("--customTag",
```

Comment:
I would use
parser.add_argument("-p", "--postfix", type=str, help="Postfix for output file name", default=None) to stay in sync with the common_parser()
then removing -p from "-p", "-pseudodata" in setupCombineWMass.py
and also removing "-a", "--append" in setupCombineWMass.py as it does the same

Reply:
makes sense, I am making the change accordingly